pkg/cluster/self_hosted.go

```diff
@@ -114,14 +114,16 @@ func (c *Cluster) migrateBootMember() error {
 		return err
 	}
 
-	delay := 20 * time.Second
-	c.logger.Infof("wait %v before removing the boot member", delay)
-	time.Sleep(delay)
+	go func() {
+		delay := 60 * time.Second
```

Comment:
@hongchaodeng I am not really sure if 60 is necessary. But let's keep it this way for safety? We can shorten this in the future if bootkube can work with shorter timeout?